starknet_py/tests/e2e/mock/contracts/oz_proxy.cairo

```diff
@@ -165,3 +161,12 @@ namespace Proxy {
         return ();
     }
 }
+
+// added for testing purposes
+@external
+func set_implementation_hash{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    new_implementation: felt
+) {
+    Proxy._set_implementation_hash(new_implementation);
+    return ();
+}
```

Comment:
I'm not sure if we should edit this file at all. We want to add some licence and these changes are probably unwanted. cc @cptartur